force-app/main/default/classes/ESA_cls_ExternalCommitteeTriggerTest.cls

```diff
@@ -0,0 +1,30 @@
+@isTest
+public class ESA_cls_ExternalCommitteeTriggerTest {
+    @TestSetup static void dataSetup() { 
+        Account acc = TestFactory.newAccount('Test Account');
+        insert acc;
+        Contact con = TestFactory.newContact('Test Contact', 'testemail@test.com', acc.Id);
+        insert con;
+        Drug__c drug = TestFactory.newDrug('testdrug');
+        insert drug;
+        External_Committee__c ec = TestFactory.newExternalCommittee('IBD DBAAC 3000-01-01', 'IBD DBAAC', acc.Id);
```

Comment:
@deepakmulamalla : This needs to be changed in the next PR, if you can change it do then please do it in a new commit.

Reply:
@cgiSunilM, just to be sure, what is the change that you are requesting here?